directories/src/redux/index.js

```diff
@@ -1,10 +1,31 @@
 import { createStore, compose, applyMiddleware } from 'redux';
+import { createLogger } from 'redux-logger';
+import { createEpicMiddleware } from 'redux-observable';
+import promise from 'redux-promise';
 import { persistReducer, persistStore } from 'redux-persist';
 import storage from 'redux-persist/lib/storage';
-import { createEpicMiddleware } from "redux-observable";
-
-import epics from "redux/epics";
 import reducer from 'redux/reducers';
+import epics from './epics';
+import { peerReceive } from './api';
+
+import { DEVELOPED_MODE } from '../config';
```

Comment:
I think there's a global variable for `__DEV__` available by default. you might not need this